transform/mattermost-analytics/models/marts/web_app/platform/fct_events_daily_snapshot.sql

```diff
@@ -0,0 +1,8 @@
+{{
```

Comment:
(nitpicking) since this table is not a real fact table, perhaps we can think a better name for the model file. 
I was thinking that since it's grouping events for `performance` category, we can use a prefix of `eg` (for event group). The rest can be something similar to `eg_[group name]s_[group table type]s.sql`. Group table type can be:
- Individual events (similar to individual transaction in a fact table).
- Periodic Snapshot (i.e. hourly, daily, weekly etc snapshot - similar to periodic snapshot fact)
- Accumulating snapshot (in case we want to track events order per user actions).

If I understand correctly, the current case is about individual events, so perhaps a nice model name would be `eg_performance_events`, assuming that the group name is `performance` and granularity is `event`.

Reply:
> This model and `int_performance_events` can be merged. It can also be updated incrementally, but this might require replacing `union_relations` with manual `UNION ALL` statement.

What is the idea behind merging these, `fct_events_daily_snapshot` has only 4 columns. `performance_events` is already a pretty huge table in terms of columns and volume.

Reply:
Got you, Thanks @ifoukarakis